cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -225,9 +225,15 @@ output SERVICE_BINDING_{{alphaSnakeUpper $name}}_ENDPOINT string = {{bicepName $
 output SERVICE_BINDING_{{alphaSnakeUpper $name}}_CONNECTION_STRING string = {{bicepName $name}}.properties.ConnectionString
 {{end -}}
 {{range $name, $value := .StorageAccounts -}}
-{{range $cname := $value.Containers -}}
+{{range $cname := $value.Blobs -}}
 output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = {{bicepName $name}}.properties.primaryEndpoints.blob
 {{end -}}
+{{range $cname := $value.Tables -}}
+output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = {{bicepName $name}}.properties.primaryEndpoints.table
+{{end -}}
+{{range $cname := $value.Queues -}}
+output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_ENDPOINT string = {{bicepName $name}}.properties.primaryEndpoints.queue
+{{end -}}
```

Comment:
Do these output names need another component so blob/table/queue do not conflict and generate the same names?

Reply:
aspire make this to be unique